src/handlers.ts

```diff
@@ -125,3 +139,62 @@ export async function handleChromiumCheck(): Promise<void> {
     throw new Error(`One or more upgrade checks failed; see the logs for details`);
   }
 }
+
+export async function handleNodeCheck(): Promise<void> {
+  let thisIsFine = true;
+  const d = debug('roller/node:handleNodeCheck()');
+  const github = await getOctokit();
+
+  d('fetching nodejs/node releases');
+  const { data: releases } = await github.repos.listReleases({
+    owner: REPOS.NODE.OWNER,
+    repo: REPOS.NODE.NAME,
+  });
+  const releaseTags = releases.map((r) => r.tag_name);
+
+  d('fetching electron/electron branches');
+  const { data: branches } = await github.repos.listBranches({
+    owner: REPOS.ELECTRON.OWNER,
+    repo: REPOS.ELECTRON.NAME,
+    protected: true,
+  });
+
+  // TODO: Implement node roller rules for release branches
+  const masterBranch = branches.find((branch) => branch.name === 'master');
+
+  d(`getting DEPS for branch ${masterBranch} in electron/electron`);
+  const depsData = await github.repos.getContents({
+    owner:  REPOS.ELECTRON.OWNER,
+    repo: REPOS.ELECTRON.NAME,
+    path: 'DEPS',
+    ref: masterBranch.commit.sha,
+  });
+  const deps = Buffer.from(depsData.data.content, 'base64').toString('utf8');
+
+  // find node version from DEPS
+  const [, depsNodeVersion] = /node_version':\n +'(.+?)',/m.exec(deps);
+  const nodeMajorVersion = depsNodeVersion.split('.')[0];
+
+  d(`computing latest upstream version for Node ${nodeMajorVersion}`);
+  const upstreamVersions =
```

Comment:
We should use the `semver` module here to parse the versions and sort them 👍 "latest" tag !== "greatest" tag